### PR TITLE
Added documentation about how to work with nested view-models

### PR DESCRIPTION
**Pull request by PatrykSzwer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,57 @@ public class BookFilter : QueryableFilter<Book>
 }
 ```
 
+### Nested view-models
```
Comment (warpech) [inline README.md]: The Table of Contents should be updated to include this new section

File: README.md [hunk 2]
```diff
@@ -221,6 +221,57 @@ public class BookFilter : QueryableFilter<Book>
...
+                        </uni-data-table-filter>
+                    </template>
+                    <template>
+                        <input type="email" value="{{item.Email.Address$::input}}" placeholder="Email">
+                    </template>
+                </uni-data-table-column>
+            </uni-data-table>
+```
+
+More about custom deifinition of the table columns you can find in the [`uni-data-table` CE documentation](https://github.com/Starcounter/uniform/tree/master/components/uni-data-table)
```
Comment (warpech) [inline README.md]: deifinition  -> definition

File: README.md [hunk 3]
```diff
@@ -221,6 +221,57 @@ public class BookFilter : QueryableFilter<Book>
...
+dataTablePage.DataTable = new DataTableBuilder<DataTableRowViewModel>()
+    .WithDataSource(DbLinq.Objects<DataTableRowDataModel>())
+    .WithColumns(columns =>
+        columns
+            .AddColumn(b => b.FirstName, column => column.DisplayName("Name"))
+            .AddColumn(b => b.Email.Address, column => column.DisplayName("Email")))
+    .Build();
+```
+
+and then add custom column display in the client side implementation:
```
Comment (warpech) [inline README.md]: I have a feeling that the client side implementation example should be not in this README, but in https://github.com/Starcounter/uniform/tree/master/components/uni-data-table (with a cross-link). WDYT?

File: README.md [hunk 4]
```diff
@@ -221,6 +221,57 @@ public class BookFilter : QueryableFilter<Book>
...
+                            Email
+                        </uni-data-table-sorter>
+                        <uni-data-table-filter value="{{column.Filter$}}">
+                        </uni-data-table-filter>
+                    </template>
+                    <template>
+                        <input type="email" value="{{item.Email.Address$::input}}" placeholder="Email">
+                    </template>
+                </uni-data-table-column>
+            </uni-data-table>
```
Comment (warpech) [inline README.md]: Inconsistent indentation

**Comment by PatrykSzwer** [comment]
I've done all requested changes, please take a look again.

**Pull request merged**